src/sentry/integrations/source_code_management/repo_trees.py

```diff
@@ -124,10 +124,12 @@ def _populate_trees(self, repositories: Sequence[dict[str, str]]) -> dict[str, R
         remaining_requests = MINIMUM_REQUESTS_REMAINING
         try:
             remaining_requests = self.get_client().get_remaining_api_requests()
-        except ApiError:
+        except Exception:
             use_cache = True
             # Report so we can investigate
-            logger.warning("Loading trees from cache. Execution will continue. Check logs.")
+            logger.warning(
+                "Loading trees from cache. Execution will continue. Check logs.", exc_info=True
```

Comment:
In this case, I want the exception to be included.

Info about `exc_info` usage:
<img width="514" alt="image" src="https://github.com/user-attachments/assets/784b74c4-46e6-448c-b050-3ab374e5f738" />